transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/packaging/ShadedJarPackaging.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.plugin.packaging;
+
+import com.github.jengelman.gradle.plugins.shadow.ShadowBasePlugin;
+import com.github.jengelman.gradle.plugins.shadow.ShadowJavaPlugin;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.linkedin.transport.plugin.Platform;
+import com.linkedin.transport.plugin.tasks.ShadeTask;
+import java.util.List;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.gradle.api.tasks.SourceSet;
+import org.gradle.api.tasks.TaskProvider;
+import org.gradle.api.tasks.bundling.Jar;
+
+import static com.linkedin.transport.plugin.ConfigurationType.*;
+import static com.linkedin.transport.plugin.SourceSetUtils.*;
+
+
+/**
+ * A {@link Packaging} class which generates Shaded JARs containing all runtime dependencies using the
+ * {@link ShadeTask}
+ * TODO: Move to using the Shade Plugin once it is open-sourced
```

Comment:
Remove.

Reply:
We should keep this here or I'm ok with a separate issue also

Reply:
Removed TODO